packages/dx-react-demos/src/bootstrap3/core-basic/pluggable-component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {
+  PluginHost,
+  Template,
+} from '@devexpress/dx-react-core';
+
+const Pluggable = () => (
```

Comment:
Can we use `export default () => (...)` or `export const Pluggable = () => (..)`? It allows to reduce code.